pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -327,10 +333,13 @@ func Run(ps *pubsub.PubSub) {
 	domainCtx.subDeviceNetworkStatus = subDeviceNetworkStatus
 	subDeviceNetworkStatus.Activate()
 
-	if err := containerd.InitContainerdClient(); err != nil {
+	if domainCtx.casClient, err = cas.NewCAS(casClientType); err != nil {
+		err = fmt.Errorf("Run: exception while initializing CAS client: %s", err.Error())
 		log.Fatal(err)
 	}
-	defer containerd.CtrdClient.Close()
+
+	//casClient which is commonly used across volumemgr will be closed when volumemgr exits.
```

Comment:
s/volumemgr/domainmgr/

Don't know if you've copied the comment to other agents as well; please check

Reply:
Yes. As of now volumeMgr and domainMgr use CAS client. Added the comment in both the services.